docs/src/main/sphinx/admin/preview-web-interface.md

```diff
@@ -0,0 +1,29 @@
+# Preview Web UI
```

Comment:
Is this the name we want to use for now .. I think yes given that the path is also `preview`.

Reply:
yes, Preview Web UI sounds good